modules/pergel.yml

```diff
@@ -3,7 +3,7 @@ description: >-
   Pergel is a tailor-made solution for Nuxt and Nitro, providing swift project
   kickstarts and seamless integration of various modules.
 repo: oku-ui/pergel#main/packages/nuxt
-npm: 'pergel'
+npm: '@pergel/nuxt'
```

Comment:
nuxt please change -> '@pergel/nuxt' -> 'pergel'

Reply:
But the Nuxt module is `@pergel/nuxt`, right?

https://github.com/oku-ui/pergel/blob/main/packages/nuxt/package.json#L2

Reply:
'pergel/nuxt' all modules are gathered into a single structure, yes, that's true, but we don't use it.

`pnpm i pergel` after add nuxt.config.ts 'pergel/nuxt'